packs/rabbitmq/config.yaml

```diff
@@ -0,0 +1,6 @@
+sensor_config:
+  poll_time: 5
+  queues:
```

Comment:
YAML also supports lists / arrays so you can use that instead of a comma delimited string.

For example:

```yaml
queues:
  - queue1
  - queue2
  -...
```